src/data/polynomial/basic.lean

```diff
@@ -84,7 +84,7 @@ they unfold around `polynomial.of_finsupp` and `polynomial.to_finsupp`.
 section add_monoid_algebra
 
 /-- The function version of `monomial`. Use `monomial` instead of this one. -/
-@[irreducible] def monomial_fun (n : ℕ) (a : R) : R[X] := ⟨finsupp.single n a⟩
+def monomial_fun (n : ℕ) (a : R) : R[X] := ⟨finsupp.single n a⟩
```

Comment:
I suspect we can now eliminate this altogether, and inline it into `monomial`.

Reply:
sure, done! had to change the definition of 1 but it seems to be defeq